clients/admin-ui/src/features/system/privacy-declarations/PrivacyDeclarationStep.tsx

```diff
@@ -54,7 +53,7 @@ const PrivacyDeclarationStep = ({ system }: Props) => {
       {isLoading ? (
         <Spinner />
       ) : (
-        <PrivacyDeclarationManager
+        <PrivacyDeclarationFormTab
```

Comment:
Can the changes in this file be safety removed?

Reply:
This change is to replace the old PrivacyDeclarationManager component in the system view with PrivacyDeclarationFormTab, which has the new forms-- the PrivacyDeclarationManager component has to still exist to appear read-only in the datamap drawer.